qualitycontrol.sh

```diff
@@ -57,6 +57,10 @@ JOBS=$((${JOBS:-1}*3/5))
 [[ $JOBS -gt 0 ]] || JOBS=1
 cmake --build . -- ${JOBS:+-j$JOBS} install
 
+if [[ $ALIBUILD_O2_TESTS ]]; then
+  make test
```

Comment:
Why not calling ctest directly ? Otherwise, if we use ninja for building, that line will fail...

Reply:
sure, I will do it.